src/lib/Form/Provider/GroupedNonMetaFormFieldsProvider.php

```diff
@@ -28,16 +29,31 @@ public function __construct(
     public function getGroupedFields(array $fieldsDataForm): array
     {
         $identifiers = $this->getMetaFields();
+        $metaFieldGroups = $this->getMetaFieldGroups();
+
+        $metaFieldIdentifiers = array_keys(
+            array_filter(
+                $fieldsDataForm,
+                static fn (FormInterface $field): bool => true
```

Comment:
Same remark on readability applies here

Reply:
@webhdx had the same idea but unfortunately, CS Fixer places it in the same line.

Reply:
Actually made it work differently :)